pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1050,19 +1050,20 @@ func (self *awsInstance) getMountDevice(volumeID string, assign bool) (assigned
 		return mountDevice(""), false, nil
 	}
 
-	// Check all the valid mountpoints to see if any of them are free
-	valid := instanceType.getEBSMountDevices()
```

Comment:
Was going to say the same thing - it makes sense to inline it because the volume list is now pretty big, but do delete getEBSMountDevices